src/test/scala/com/gu/contentapi/LambdaSpec.scala

```diff
@@ -79,14 +80,62 @@ class LambdaSpec extends FlatSpec with Matchers {
     thirdPw should be(pageViews(2))
   }
 
-  // Commented out because we don't want to run CAPI query on TeamCity.
+  // The following tests are commented out because we don't want to run CAPI query on TeamCity,
```

Comment:
Why not use them as integration tests which can be run with `sbt it:test` like we do in Bonobo: https://github.com/guardian/bonobo/blob/master/build.sbt#L7

That way they don't need to be commented out and can be run by the developer without them being run each time we do a build.